src/components/Table/TableBody/TableBodyRow/TableBodyRow.story.jsx

```diff
@@ -43,6 +43,21 @@ storiesOf('Watson IoT|TableBodyRow', module)
       options={{ hasRowActions: true, hasRowExpansion: true }}
     />
   ))
+  .add('is not selectable', () => (
+    <TableBodyRow
```

Comment:
Checkboxes here aren't aligned properly, though they appear correct in the StatefulTable story both in unselectable and selectable state.

![image](https://user-images.githubusercontent.com/3360588/66329876-e8635900-e8f4-11e9-962a-8b99673412de.png)

Reply:
This is probably related https://github.com/IBM/carbon-addons-iot-react/pull/592/files?file-filters%5B%5D=.jsx#r332114465

Reply:
this ones does actually appear to be broken in the underlying stateful table as well.  I think this is due to the wrong checkbox being used.